students/charlie_england/session02/grid_printer.py

```diff
@@ -10,4 +10,14 @@ def pri_grid(size = 2, cols = 2, rows = 2): #instead of just 2 parameters, allow
             print((" |" + "  "*size)*cols+" |")
     print((" +" + " -"*size)*cols+" +") 
 
-pri_grid(5,5,10)
\ No newline at end of file
+pri_grid(5,5,10)
+=======
```

Comment:
this is an artifact of the merge conflict -- you'll get a syntax error if you try to run the code.

As a rule, you want to test your code after "resolving the conflict", before committing the change.

Reply:
I'm going to merge, so we can move on.

Whether you want to fix this now is up to you -- you've gotten credit for the assignment!

Reply:
Thanks! still getting used to github and I edited it inside of github. Next time I'll do the git pull upstream master and edit it inside my IDE and make sure to test it. Appreciate the help!